modules/web-console/frontend/app/components/connected-clusters-dialog/components/list/column-defs.ts

```diff
@@ -48,7 +48,7 @@ export const columnDefsFn = ($translate: ng.translate.ITranslateService) => [
                     value='COL_FIELD'
                 ></connected-clusters-cell-status>
                 <connected-clusters-cell-logout
-                    ng-if='row.entity.secured && grid.appScope.$ctrl.agentMgr.hasCredentials(row.entity.id)'
+                    ng-if='row.entity.secured'
```

Comment:
Looks like that this `ng-if` may not work as expected after this change.

Reply:
All works fine.